### DISCUSSION TITLE
使用并行计算的一点经验教训 | /project/glmnet-parallel/

### DISCUSSION THREAD

**Discussion by giscus** [earfanfan/yuanfan#643] [Announcements]
https://yuanfan.rbind.io/project/glmnet-parallel/

**Comment by xiaole12138** [comment]
作者有机会的话可以再讲讲并行计算的内容吗？最近在学并行计算，parLapply那些函数总是不会用。另外想问一下作者现在是在Linux系统下用Rstudio吗？运行速度相比于在Windows下快吗？

**Reply by earfanfan** [reply]
作者可能没机会再讲讲了，作者很久没接触过并行计算了。不过如果你在学习的过程中遇到问题，可以把具体的问题整理一下拿去统计之都论坛问问<d.cosx.org>，我当时学的时候也是在那问了很多。

我在 Linux 系统下用的 RStudio 实际上用的是服务器版本的 RStudio Server，因为硬件条件好，所以快些。

**Reply by xiaole12138** [reply]
好的，感谢
